content/docs/guides/observability/metrics.md

```diff
@@ -165,11 +167,15 @@ prometheus.io/port: 15010
 prometheus.io/path: /stats/prometheus
 ```
 
-### Available Metrics
+When metrics are disabled for a previously enabled meshed namespace, the existing pods in the namespace will still configured for scraping. The pods must be recreated for the annotation to be removed. All future pods added to a namespace with metrics disabled will not be updated with the Prometheus annotations.
+
+## Available Metrics
 
-For details about what metrics are scraped from each Envoy proxy, see [Envoy's documentation](https://www.envoyproxy.io/docs/envoy/v1.17.2/operations/stats_overview). Note that OSM's default configuration only scrapes a subset of all metrics generated by each proxy.
+OSM exports metrics about the traffic within the mesh as well as metrics about the control plane.
 
-#### Custom Metrics
```

Comment:
I'm inclined to keep this heading as "Custom Metrics" (or "Custom Envoy Metrics") since this section only describes the metrics OSM adds to Envoy, not all of the Envoy metrics available.

Reply:
Updated to "Custom Envoy Metrics"